samcli/cli/command.py

```diff
@@ -19,6 +19,7 @@
     "samcli.commands.local.local",
     "samcli.commands.package",
     "samcli.commands.deploy",
+    "samcli.commands.delete",
```

Comment:
We need to add it to pyinstaller hook file: https://github.com/aws/aws-sam-cli/blob/develop/installer/pyinstaller/hook-samcli.py

Reply:
done